src/Classes/Loader.ts

```diff
@@ -35,7 +35,7 @@ export class Loader {
      * @param params the options array to get the item from
      * @param index if provided gets the corresponding item from the array
      */
-    public static loadFromArray(tagId: string, params: IOptions[], index?: number): Container | undefined {
+    public static loadFromArray(tagId: string, params: RecursivePartial<IOptions>[], index?: number): Container | undefined {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5179066)